packages/cozy-harvest-lib/package.json

```diff
@@ -29,12 +29,12 @@
     "@babel/runtime": "^7.5.2",
     "@material-ui/core": "3",
     "cozy-doctypes": "^1.69.0",
```

Comment:
Why do you add UI as a dep? It should remains only dev and peerdep nop?

Reply:
Fixed, thanks